content/java/concepts/math-methods/terms/next-up/next-up.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.nextUp()'
+Description: 'Returns the floating-point value adjacent to the argument in the direction of positive infinity.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.nextUp()`** method returns the floating-point value adjacent to the argument passed as the parameter in the direction of positive infinity. Argument passed can be either of type `double` or `float`.
```

Comment:
Move "Argument passed can be either of type `double` or `float`." to the "Syntax" section, then you can have one ```pseudo block reading `Math.nextUp(val)`.